Applications/CMC/test/testCMCGait10dof18musc.cpp

```diff
@@ -80,8 +81,12 @@ void testGait10dof18musc() {
 
     int nstates = standard->getColumnLabels().size() - 1;
 
-    // angles and speeds within .6 degrees .6 degs/s; activations within 1%
-    std::vector<double> rms_tols(nstates, 0.01);
+    // angles and speeds within 0.01 rads and 0.1 rad/s;
+    // and activations to within 10%
```

Comment:
Seems okay to increase tolerance for activations from one arbitrary value (1%) to another arbitrary value (10%) for now. In the future, I recommend splitting these tests in twain:
1. Regression test: ensure the computed values are exactly as expected (to within some reasonable but small tolerance).
2. Logical test: ensure each muscle is on/off at the expected times.